Directory.Packages.props

```diff
@@ -1,4 +1,6 @@
 <Project>
+  <!-- Workaround https://github.com/dependabot/dependabot-core/issues/8490 -->
+  <Import Project="eng/Versions.props" Condition="'$(MajorVersion)' == ''"/>
```

Comment:
Yeah, typically when doing this a new property would get added to Versions.props called something like `VersionPropsAlreadyImported` or something like that, and then that is what is checked here. Mainly in case `MajorVersion` gets defaulted somewhere else or moved which might cause issues in the future.

Reply:
Yeah, it's temporary and if that happens there'll be 200 warnings so we'll know!